project/source/game/Game.cpp

```diff
@@ -304,19 +304,19 @@ void Game::OnTick(float dt)
 	// check for memory corruption
 	assert(_CrtCheckMemory());
 
-	if(dt > LIMIT_DT)
+	if (dt > LIMIT_DT)
```

Comment:
ustaw sobie żeby nie stawiał spacji w "if (" bo potem będziesz miał 500 zmian 😜

Reply:
a to źle jak jest spacja po if? podobno tak powinno się stawiać

Reply:
[https://stackoverflow.com/questions/4368055/coding-style-advice-rationales-for-placing-spaces-in-control-statements-with-c](url)

W skrócie: kwestia gustu :P a że w tym projekcie jest taki standard to nie zmieniaj (szczególnie jak sobie ustawisz autoformatowanie pliku przy zapisie to ci wszystko przeora).

Reply:
Dorzuć do projektu .clang-format i tego się trzymaj :P

Reply:
no właśnie - nie powinien się taki plik ustawień ładować wraz z projektem? inaczej skąd mam wiedzieć jakie ustawienia formatowania zastosować?

Reply:
@Leinnan testowałem ten clang-format ale nie da się niestety tego ustawić tak żeby nieprzeorał 50% projektu. np.: https://stackoverflow.com/questions/33656800/clang-format-line-breaks/34339430#34339430